Darwin/Services/StitchLocalMongoDBService/StitchLocalMongoDBService/LocalMongoClient.swift

```diff
@@ -61,21 +61,21 @@ private final class MobileMongoDBClientFactory: CoreLocalMongoDBService, Throwin
         NotificationCenter.default.addObserver(
             self,
             selector: #selector(batteryLevelDidChange(_:)),
-            name: .UIDeviceBatteryLevelDidChange,
+            name: UIDevice.batteryLevelDidChangeNotification,
```

Comment:
[q] does this affect backwards compatibility?

Reply:
Not for users, but it does for developers.